packages/sw-routing/test/static/route.js

```diff
@@ -9,38 +9,29 @@ const routes = [];
 
 routes.push(new goog.routing.Route({
   match: ({url}) => url.pathname.endsWith('static'),
-  handler: {
-    handle: () => Promise.resolve(new Response('static response')),
-  },
+  handler: () => Promise.resolve(new Response('static response')),
 }));
 
 routes.push(new goog.routing.Route({
   match: ({url}) => url.pathname.endsWith('method/put'),
-  handler: {
-    handle: () => Promise.resolve(new Response('put response')),
-  },
+  handler: () => Promise.resolve(new Response('put response')),
   method: 'PUT',
 }));
 
 routes.push(new goog.routing.Route({
   match: ({url}) => url.pathname.endsWith('echobody'),
-  handler: {
-    handle: ({event}) => event.request.text()
-      .then((body) => new Response(body)),
-  },
+  handler: ({event}) => event.request.text().then((body) => new Response(body)),
   method: 'POST',
 }));
 
 routes.push(new goog.routing.Route({
   match: ({url}) => url.pathname.match(
     new RegExp('/echo3/1st/(\\w+)/2nd/(\\w+)/3rd/(\\w+)')),
-  handler: {
-    handle: ({params}) => Promise.resolve(
-      new Response(JSON.stringify(params.slice(1)), {
-        headers: {'content-type': 'application/json'},
-      })
-    ),
-  },
+  handler: ({params}) => Promise.resolve(new Response(
+    JSON.stringify(params.slice(1)), {
+      headers: {'content-type': 'application/json'},
+    })
+  ),
```

Comment:
It's [covered in the unit tests](https://github.com/GoogleChrome/sw-helpers/pull/261/files#diff-9e4ba02317a767b390dca5110a56ed26R38), so I figured it was safe to stick with one approach instead of covering all variations within the end-to-end tests.